const TodoTask = require('../models/TodoTask');const errorHandler = require('../utils/errorHandler');const {update} = require("./todo");const User = require("../models/User");module.exports.getByCategoryId = async function (req, res) {    try {        const todo = await TodoTask.find({            todo: req.params.todoId,            user: req.user.id        });        res.status(200).json(todo)    } catch (e) {        errorHandler(res, e)    }};module.exports.remove = async function (req, res) {    try {        await TodoTask.remove({_id: req.params.id});        res.status(200).json({            message: 'Категория удалена.'        })    } catch (e) {        errorHandler(res, e)    }};module.exports.update = async function (req, res) {    const updated = {        name: req.body.name    };    try {        const todo = await TodoTask.findOneAndUpdate(            {_id: req.params.id},            {$set: updated},            {new: true}        );        res.status(200).json(todo)    } catch (e) {        errorHandler(res, e)    }};module.exports.create = async function (req, res) {    console.log(req.user);    console.log(req.body);    const todo = new TodoTask({        name: req.body.name,        user: req.user.id,    });    try {        await todo.save();        res.status(201).json(todo)    } catch (e) {        errorHandler(res, e)    }};